Cancel rift

uofirob opened the pull request:
1) Bug fix for creating rifts with spaces (Ant Man, Giant Man, Yellow Jacket)
2) Add functionality for cancelling rift (need to test, I couldn't test in my mock-only setup)
3) I need you to create the unit tests for cancelling the rift successfully (At almost midnight, I couldn't wrap my head about mocking everything out.  I need to mock create a rift and then mock cancel it, but not sure how to get the ids that would be necessary to do that)

uofirob committed:
Preliminary Changes for Rift Cancel

First changes + Fix for rifts with spaces in the name

uofirob committed:
Fix tests for Reifts with spaces

uofirob committed:
Merge branch 'master' of https://github.com/chrischen86/Hudson into Cancel_Rift

uofirob committed:
Add in cancel rift functionality

Update  to add in cancel rift functionality (first pass)

uofirob committed:
Incorporate requested changes

chrischen86 reviewed (changes requested):
Can you change the PR to merge into riftCancel?

Also marking is_deleted in rift_history should be enough; if we want to delete slack message history we would run into FK problems with rift_history.slack_message_id

uofirob committed:
Check in requested changes

Remove deletion of the rift message from history (need to use slack message api)
